Formula/influxdb.rb

```diff
@@ -58,6 +58,51 @@ def install
     ldflags = "-s -w -X main.version=#{version}"
     system "go", "build", *std_go_args(ldflags: ldflags), "-o", bin/"influx", "./cmd/influx"
     system "go", "build", *std_go_args(ldflags: ldflags), "-tags", "assets", "-o", bin/"influxd", "./cmd/influxd"
+
+    # Generate default config file.
+    # TODO: Warn if there's already a config file there?
```

Comment:
@carabolic how do formulas that generate/write default configs usually handle this?

Reply:
make a postinstall block and do `action unless File("").exists?`